dev/snippets/lib/snippets.dart

```diff
@@ -127,15 +127,19 @@ class SnippetGenerator {
       'description': description,
       'code': htmlEscape.convert(result.join('\n')),
       'language': language ?? 'dart',
-    }..addAll(type == SnippetType.application
-        ? <String, String>{
-            'serial': metadata['serial'].toString() ?? '0',
-            'id':
-                injections.firstWhere((_ComponentTuple tuple) => tuple.name == 'id').mergedContent,
-            'app':
-                htmlEscape.convert(injections.firstWhere((_ComponentTuple tuple) => tuple.name == 'app').mergedContent),
-          }
-        : <String, String>{'serial': '', 'id': '', 'app': ''});
+      if (type == SnippetType.application)
```

Comment:
Nit - this isn't really any worse than the previous construction, but still looks awkward to me.  I wonder if it would read better to just initialize these to their empty values, and below do an `if (type == SnippetType.application) { // update the values }`

Reply:
Done